src/redux/userInput/userInputSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const userInputSlice = createSlice({
+  name: 'userInput',
+  initialState: { description: '', location: '' },
```

Comment:
description e location in questo caso ci stanno su redux, ma in un contesto reale con molte piu azioni, non ha senso spammare redux per ogni tasto digitato, meglio gestire queste come stato interno del componente e mettere su redux lo stato che puo' interessare ad N altre pagine o componenti (tipo i valori che sono stati cercati)